<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="sidePanel">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="sidePanelContent">
       <layout class="QVBoxLayout" name="sidePanelLayout"/>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="taskComboBox"/>
    </item>
    <item>
     <widget class="QWidget" name="mainPanel">
      <layout class="QVBoxLayout" name="mainPanelLayout"/>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="submitButton">
      <property name="text">
       <string>Assign</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
